### ISSUE TITLE
Cross compile guide

### ISSUE THREAD

**Issue by empirefox** [TokTok/c-toxcore#861]
I've tried dist-build/android-armv7.sh, but got no luck.

**Comment by sudden6** [comment]
These scripts are completely outdated and not maintained anymore. cc: @zoff99

**Comment by zoff99** [comment]
@empirefox do you want to compile for android?

**Comment by empirefox** [comment]
@zoff99 yes. I want to compile with golang. For now I just succeed compiling c-toxcore using the way of [#507](https://github.com/TokTok/c-toxcore/issues/507). But I cannot find a way to compile using [gomobile bind](https://github.com/golang/mobile).

**Comment by zoff99** [comment]
i dont know anything about golang.

**Comment by empirefox** [comment]
Thanks @zoff99. looks like i must do the experiments by myself.

**Comment by sudden6** [comment]
@empirefox maybe you can get some inspiration from the qTox cross-compile script for windows, https://github.com/qTox/qTox/blob/master/windows/cross-compile/build.sh#L895-L950
